app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -271,6 +271,21 @@ input.search-bar {
   }
 }
 
+.btn-link {
+  background-color: transparent;
+  border: 0;
+  color: #0d6efd;
```

Comment:
I think this should match the PrimaryColor

Reply:
It's to match the Powered by Greenlight link. IMO It has to be blue so it indicates it is a link.

Reply:
Yeah but we're branding everything else, so we might as well brand the links as well

Reply:
We can discuss this tomorrow

Reply:
I'd keep the links blue. But yeah I know what you mean. I have been branding the inputs for example but I think we should let it the default blue as well.

Reply:
Yeah we could brand it too it looks all right.